packages/core/core-flows/src/definition/cart/workflows/update-line-item-in-cart.ts

```diff
@@ -92,10 +92,13 @@ export const updateLineItemInCartWorkflow = createWorkflow(
 
     parallelize(
       refreshCartShippingMethodsStep({ cart }),
```

Comment:
Not related, but promotions step is kinda dependent on this completing first as we use the shipping methods as a part of the context. Parallelising this would lead to a temporary wrong result.

Reply:
yeah, good point.
moved them to outside the parallel as well.